src/app/shared/code-editor/code-editor.component.ts

```diff
@@ -91,6 +93,16 @@ export class CodeEditorComponent implements AfterViewInit {
         sampleCodeUrl = '../assets/text/sample.nf';
       } else if (this.editorFilepath === '/nextflow.config') {
         sampleCodeUrl = '../assets/text/nextflow.config';
+      } else if (this.mode === 'yaml') {
+        if (this.entryType === EntryType.BioWorkflow) {
+          sampleCodeUrl = '../assets/text/sample-workflow-dockstore.yml';
+        } else if (this.entryType === EntryType.AppTool || this.entryType === EntryType.Tool) {
```

Comment:
Actually, it appeared that it was the opposite, it only ever happened for a `Tool` and not an `AppTool` because we were passing `Tool` to the register github app component. Fixed it so now it should only ever be `AppTool`